src/main/java/org/bahmni/mart/exports/Form2ObservationExportStep.java

```diff
@@ -136,7 +131,7 @@ private String getFormName() {
     public Step getRemovalStep() {
         return stepBuilderFactory.get(getStepName("Removal Step"))
                 .<Map<String, Object>, Map<String, Object>>chunk(100)
-                .reader(obsReader(true))
+                .reader(obsReader())
```

Comment:
Removal step supposed to remove voided records only. Is it doing same?

Reply:
Discussed with Siva